templates/admin/citizen_project/list_committee_supports.html.twig

```diff
@@ -6,7 +6,7 @@
             {% for committeeSupport in object.committeeSupports %}
                 <li>
                     {{ committeeSupport.committee.name }}
-                    [{{ committeeSupport.status }}]
+                    [{{ committeeSupport.committee.status }}]
```

Comment:
🤔 we want to display `CitizenProjectComitteeSupport::$status` here, not the Comittee's status.

Reply:
Ok so I add the `status` getter in `CitizenProjectComitteeSupport`, because there was a bug here.